app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+    def __repr__(self) -> str:
+        return str(self.__dict__)
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        if person.get("wife"):
+            Person.people.get(person.get("name")).wife = (
+                Person.people.get(person.get("wife")))
```

Comment:
bad code style, fix it. Here is an example: https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md#code-style